mlrun/runtimes/daskjob.py

```diff
@@ -407,10 +408,14 @@ def deploy(
             show_on_failure=show_on_failure,
         )
 
+    # TODO: Remove in 1.5.0
+    @deprecated(
+        version="1.3.0",
+        reason="'Dask gpus' will be removed in 1.5.0, use 'with_limits' instead",
+    )
     def gpus(self, gpus, gpu_type="nvidia.com/gpu"):
-        raise NotImplementedError(
```

Comment:
To not confuse reviewers - this was removed by me in development (1.3.0) and I later saw it in the ticket